setup.py

```diff
@@ -21,7 +22,7 @@
         'Programming Language :: Python :: Implementation :: CPython',
         'Topic :: Scientific/Engineering :: Artificial Intelligence',
         'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 3.7'
+        'Programming Language :: Python :: 3.9'
       ],
       keywords='worker',
       url='https://github.com/iterait/shepherd',
```

Comment:
Je potřeba tady změnit nějaké info ve smyslu "Iterait -> Mild Blue"? Třeba přidat do authorů, změnit url atd?

Reply:
asi to pak stejne cely uzavreme a nebudeme to davat public, jen musim kouknout jak to spravne udelat, takze neres, zatim bych to nedelal.